### PR TITLE
tools: update cluster binlog and sync-diff-inspector

### PR DESCRIPTION
**Pull request by lilin90**
@WangXiangUSTC @liubo0127 @CaitinChen 
Via: https://github.com/pingcap/docs-cn/pull/1046

### CODE REVIEW

**Comment by WangXiangUSTC** [comment]
LGTM

File: tools/tidb-binlog-cluster.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ The server hardware requirements for development, testing, and the production en
...
 * If you want to start Drainer in the existing TiDB cluster, generally, you need to make a full backup of the cluster data, obtain `savepoint`, import the data to the target database, and then start Drainer to synchronize the incremental data from `savepoint`.
 * Drainer supports synchronizing binlogs to MySQL, TiDB, Kafka or the local files. If you need to synchronize binlogs to other destinations, you can set Drainer to synchronize the binlog to Kafka and read the data in Kafka for customization processing. See [Binlog Slave Client User Guide](../tools/binlog-slave-client.md).
-* If TiDB-Binlog is used for recovering the incremental data, you can set the downstream to `pb` to synchronize the binlog data to the local file and then use [Reparo](../tools/reparo.md) to recover the incremental data.
+* If TiDB-Binlog is used for recovering the incremental data, you can set the downstream to `pb` (local files in the proto buffer format). Drainer converts the binlog to data in the specified proto buffer format and writes the data to local files. In this way, you cam use [Reparo](../tools/reparo.md) to recover the incremental data.
```
Comment (CaitinChen) [inline tools/tidb-binlog-cluster.md]:
```suggestion
* If TiDB-Binlog is used for recovering the incremental data, you can set the downstream to `pb` (local files in the proto buffer format). Drainer converts the binlog to data in the specified proto buffer format and writes the data to local files. In this way, you can use [Reparo](../tools/reparo.md) to recover the incremental data.
```

**Comment by lilin90** [comment]
@WangXiangUSTC @CaitinChen I added the relationship between TiDB-Binlog and TiDB. PTAL again~ Thanks!

**Comment by WangXiangUSTC** [comment]
LGTM

**Pull request merged**